packages/react-cosmos/src/server/server.js

```diff
@@ -42,6 +43,11 @@ export default function startServer() {
   const loaderCompiler = webpack(loaderWebpackConfig);
   const app = express();
 
+  if (httpProxy) {
+    const { context, target } = httpProxy;
```

Comment:
How about an informative log like we do for `publicPath` at line 65?

Reply:
This is already built in to the middleware library:
```
[Cosmos] No webpack config found, using defaults
[HPM] Proxy created: /api  ->  http://localhost:4001
[Cosmos] See you at http://localhost:8989/
```